apps/web/hooks/useLinkResolver/useLinkResolver.ts

```diff
@@ -194,6 +194,18 @@ export const routesTemplate = {
     is: '/[slug]',
     en: '/en/[slug]',
   },
+  universitysearchdetails: {
+    is: '/haskolanam/[id]',
+    en: '/en/haskolanam/[id]',
```

Comment:
Should the english path be in english? Such as `/en/uni-education/[id]`?

Reply:
I do believe the final url for this project has yet to be decided so for the time being it can be like this, this isn't going live for a while, this is just for testing purposes